enclave-runtime/src/tests.rs

```diff
@@ -466,17 +504,25 @@ fn test_top_pool() -> TestTopPool {
 /// Decrypt `encrypted` and decode it into `StatePayload`
 fn state_payload_from_encrypted(encrypted: &[u8]) -> StatePayload {
 	let mut encrypted_payload: Vec<u8> = encrypted.to_vec();
-	AesSeal::unseal()
-		.map(|key| key.decrypt(&mut encrypted_payload))
-		.unwrap()
-		.unwrap();
+	let state_key = state_key();
+	state_key.decrypt(&mut encrypted_payload).unwrap();
 	StatePayload::decode(&mut encrypted_payload.as_slice()).unwrap()
 }
 
+fn state_key() -> Aes {
+	Aes::default()
+}
```

Comment:
Use a default AES key for state encryption instead of reading it from file (and thus requiring that file to exists when we run the tests)